halo/config/config.go

```diff
@@ -25,9 +25,10 @@ const (
 	networkFile     = "network.json"
 	attestStateFile = "xattestations_state.json"
 
-	DefaultHomeDir          = "./halo" // Defaults to "halo" in current directory
-	defaultSnapshotInterval = 1000     // Roughly once an hour (given 3s blocks)
-	defaultMinRetainBlocks  = 0        // Retain all blocks
+	DefaultHomeDir            = "./halo" // Defaults to "halo" in current directory
+	defaultSnapshotInterval   = 1000     // Roughly once an hour (given 3s blocks)
+	defaultSnapshotKeepRecent = 2
```

Comment:
Do you think having the same defaultSnapshotKeepRecent in the main net is safe? is there any. scenario in which having more snapshots might help. May be when there is a stall or something..

Reply:
this is the default of cosmos-SDK. I assume two is good enough. Basically just latest, and a backup of latest.